src/background/onboarding.js

```diff
@@ -0,0 +1,35 @@
+import browser from "webextension-polyfill";
+import SLStorage from "../popup/SLStorage";
+
+let POST_SETUP_URL = null;
+
+function initService() {
```

Comment:
should we add `async` for `initService` as it uses `listenPostSetup` which is an async function?

Reply:
It's not necessary, because `async` is required only when `await` is used. For now, we don't use `await` because we don't need to wait for `listenPostSetup`

Reply:
as PyCharm always flags this as a warning, I wonder if this is a best practice to always use async for a function that calls another async function? This is enforced in Python as well (the code doesn't work if async function is called inside a non-async function).